src/RedialContext.js

```diff
@@ -188,7 +192,10 @@ export default class RedialContext extends Component {
 
     Promise.all(promises)
       .then(this.props.onCompleted)
-      .catch((err) => this.props.onError(err, bail() || 'other'));
+      .catch((err) => {
+        this.props.onError(err, bail() || 'other');
+        this.abort(true);
+      });
```

Comment:
Can you make an example where for one route, the blocking fetch fails, and for another route the deferred fetch fails. I think we might want to handle these differently...